examples/hybrid/sphere/balanced_flow_rhoe.jl

```diff
@@ -1,25 +1,20 @@
 using Test
 using ClimaCorePlots, Plots
-using DiffEqCallbacks
-using JLD2
 
 include("baroclinic_wave_utilities.jl")
 
-jld2_callback = PeriodicCallback(output_writer, 864000; initial_affect = true)
+const sponge = false
 
-driver_values(FT) = (;
-    zmax = FT(30.0e3),
-    velem = 10,
-    helem = 4,
-    npoly = 4,
-    tmax = FT(60 * 60),
-    dt = FT(5.0),
-    ode_algorithm = OrdinaryDiffEq.SSPRK33,
-    jacobian_flags = (; ∂𝔼ₜ∂𝕄_mode = :no_∂P∂K, ∂𝕄ₜ∂ρ_mode = :exact),
-    max_newton_iters = 2,
-    save_every_n_steps = 10,
-    additional_solver_kwargs = (;), # callback = jld2_callback), 
+space = ExtrudedSpace(;
+    zmax = FT(30e3),
+    zelem = 10,
+    hspace = CubedSphere(; radius = R, pelem = 4, npoly = 4),
 )
+tend = FT(60 * 60)
```

Comment:
I have a slight preference of `tmax` or `t_end` because `tend` sounds like tendency to me...but up to you

Reply:
Okay yeah, I also prefer `t_start` and `t_end` to `tstart` and `tend`.